### PR TITLE
Initial tool review

### PR DESCRIPTION
**Pull request by simonbray**
Starting a review of all the tools in this repository, as we discussed at GCC. These are some initial changes, mostly spelling and other cosmetic fixes. Will make some more substantial changes in other PRs.

Don't merge this straight away though, I don't want to bump the version numbers in every PR...

### CODE REVIEW

**Comment by simonbray** [comment]
https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/issues/31
https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/issues/36

**Comment by simonbray** [comment]
@chrisbarnettster @tsenapathi @bgruening any comments on this? I think it's almost ready to merge (though I might like to add the changes to the gromacs wrappers too https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/pull/37).
The changes are not that drastic: removing some typos, padding out the help sections a little, bumping version numbers, adding gromacs file types to the MDAnalysis tools (could not add them to the bio3d tools unfortunately).

**Comment by bgruening** [comment]
Good for me, please add all your changes.

File: tools/buildtools/packmol/packmol.xml [hunk 1]
```diff
@@ -69,11 +69,11 @@ with open("$parameteroutfile",'w+') as f:
                 <expand macro="multiple_input" name="packmol_inputs" format="moldy" label="moldy input with molecule type" help_text="Select a moldy file in .moldy format"/>
             </when-->
         </conditional>
-        <section name="allparams" title="General Parameters" expanded="false">
-            <param name="tolerance" type="float" value="2.0" label="distance tolerance" min="0.0" max="100.0"/>
-            <param name="nloop" type="integer" value="20" label="maximum number of optimisation loops" min="1" max="10000"/>
-            <param name="maxit" type="integer" value="20" label="maximum number of gencan iterations per loop" min="1" max="10000"/>
-            <param name="seed" type="integer" value="-1" label="random number generator seed, default of -1 used the computer time to set the seed" />
+        <section name="allparams" title="General parameters" expanded="false">
```
Comment (simonbray) [inline tools/buildtools/packmol/packmol.xml]: @chrisbarnettster can we remove this conditional for now until/if the other formats are added? It looks a bit strange from a UI perspective.

File: tools/mdanalysis/rdf.xml [hunk 2]
```diff
@@ -24,15 +26,15 @@
...
-        <param name="segid2"  type="text" value="HET" label="Segid of atom 2"/>
-        <param name="resid2"  type="text" value="3" label="Resid of atom 2"/>
+        <param name="segid2"  type="text" value="HET" label="Segment ID of atom 2"/>
+        <param name="resid2"  type="text" value="3" label="Residue ID of atom 2"/>
         <param name="name2"  type="text" value="C1" label="Atom name of atom 2"/>
         <param name="nbins"  type="text" value="100" label="Number of bins in the histogram"/>
-        <param name="start"  type="text" value="0.0" label="Starting Point"/>
-        <param name="end"  type="text" value="5.0" label="End Point"/>
+        <param name="start"  type="text" value="0.0" label="Starting point"/>
+        <param name="end"  type="text" value="5.0" label="End point"/>
```
Comment (simonbray) [inline tools/mdanalysis/rdf.xml]: @tsenapathi @chrisbarnettster I really struggled to understand the purpose of these start/end params. Is it supposed to allow the user to just study a section of the trajectory?

Comment (chrisbarnettster) [reply tools/mdanalysis/rdf.xml]: Yes. Either to remove initial section, test on first portion, etc. 
These are optional params though https://www.mdanalysis.org/docs/documentation_pages/analysis/rdf.html and the behaviour should probably be updated.

File: tools/buildtools/packmol/packmol.xml (hunk 1 above)
Comment (chrisbarnettster) [reply tools/buildtools/packmol/packmol.xml]: Hi @simonbray 
sure, some minor updates to templates will be needed. I'm happy to leave the commented placeholder code for the future 😄

File: tools/mdanalysis/rdf.xml (hunk 2 above)
Comment (simonbray) [reply tools/mdanalysis/rdf.xml]: But this kind of option could be useful for all tools, right? Not just RDF.

Comment (chrisbarnettster) [reply tools/mdanalysis/rdf.xml]: yes definitely! I think by default should be 1 to the end but useful to allow this kind of functionality. 
@tsenapathi any feedback?

Comment (simonbray) [reply tools/mdanalysis/rdf.xml]: Or even have a separate tool to cut slices out of trajectories?

File: tools/buildtools/packmol/packmol.xml (hunk 1 above)
Comment (simonbray) [reply tools/buildtools/packmol/packmol.xml]: done in https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/pull/35/commits/6fa3234ddf93f4fbda7e54c19a192c8a3c1fe58e

File: tools/mdanalysis/rdf.xml (hunk 2 above)
Comment (chrisbarnettster) [reply tools/mdanalysis/rdf.xml]: Separate tool could be mdconvert (part of mdtraj package). Supports striding and slice notation for frame selection.

Comment (simonbray) [reply tools/mdanalysis/rdf.xml]: Cool, looks good. I'll make the tool then. Then we don't need to make an extra option for all the tools.

Comment (simonbray) [reply tools/mdanalysis/rdf.xml]: Added a wrapper. Strangely, each time I slice a dcd file in the same way (on the command line) I get a different binary file, so had to use `sim_size` for the test. Not a problem for an xtc file.

Comment (simonbray) [reply tools/mdanalysis/rdf.xml]: After rereading the mdanalysis documentation carefully I realized I misunderstood the nature of these parameters. They define a range of r over which the RDF is calculated. The slicing tool was still worth making though.

**Comment by simonbray** [comment]
I think this can be merged.

**Comment by simonbray** [comment]
yes, done

**Comment by bgruening** [comment]
Merci!

**Comment by simonbray** [comment]
Thanks Björn

**Pull request merged**